### DISCUSSION TITLE
Facing issue while opening PDF From github repository

### DISCUSSION THREAD

**Discussion by vishnupriya2503** [community/community#76403] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

I have been given access to github repository of my co worker to access project file 

I am unable to access the PDF File from that repository for some reason but i can able to open text file from the same repository.

While opening the PDF File i am receiving Error as "Unable to render code block"

Note : i tried clearing the browser cache and extension but still facing the same issue.
![github PDF issue](https://github.com/community/community/assets/150783105/dfc9e029-46a3-4d25-83d1-85b8fcf5993b)

**Comment by subidit** [comment]
It happens for large PDF files. 
You would be better off to just download the file. There is an option to download file on the top right corner (shortcut `ctrl shift s`)

You could also clone the whole repository in your local machine and use `git pull` to keep updated with any future changes.

**Comment by hello-Rahul24** [comment]
hey Pal, i am also facing the same problem , did you find any solution ?? please let me know.
